specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/services.json

```diff
@@ -500,6 +554,16 @@
         }
       }
     },
+    "MaterializedViewsBuilderServiceResourceCreateUpdateParameters": {
```

Comment:
Description has to change to reflect MaterializedViewsBuilderServiceResourceCreateUpdateParameters.